### DISCUSSION TITLE
CRD does not seem to be a injectable resource for the cert-manager

### DISCUSSION THREAD

**Discussion by Iceber** [vmware-tanzu/velero#4177] [Community support Q&A] [answered]
I've observed a lot of strange annotations in crd, is there any special meaning to this?
https://github.com/vmware-tanzu/velero/blob/c21b66166a66a4f481712fd5b29a7cbc5dfc0b61/test/e2e/testdata/enable_api_group_versions/case-b-source-manually-added-mutations.yaml#L1-L7

`cainjector` only helps to configure the CA certificate for webhooks. It doesn't seem to be of any use to crd.

**Comment by zubron** [comment, marked as answer]
These CRDs were originally taken from another project where there were webhooks in use and the annotations were relevant there (if you are interested, see https://github.com/vmware-tanzu/velero/tree/main/test/e2e/testdata/enable_api_group_versions and https://github.com/brito-rafa/k8s-webhooks/blob/master/examples-for-projectvelero/README.md). These CRDs are only used as test data to validate backup/restore behaviour and are not used by the main Velero project.

**Reply by Iceber** [reply]
Thanks! 😃
